Library/Formula/hashcat.rb

```diff
@@ -0,0 +1,39 @@
+class Hashcat < Formula
+  desc "Advanced CPU-based password recovery utility"
+  homepage "https://hashcat.net/hashcat/"
+  url "https://codeload.github.com/hashcat/hashcat/tar.gz/2.00"
+  sha256 "6325e6d75a4df3485adec00f74e5887326809c15ed31bfe74a12b62943245444"
+
+  head do
+    url "https://github.com/hashcat/hashcat.git"
+  end
+
+  depends_on "lzip"
+  depends_on "gcc" # with multilib
```

Comment:
Upstream says it needs GCC with multilib, which is the default in homebrew's GCC formula. I don't know if/how it should be described here.